sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -121,11 +121,15 @@ waitUntilBuilt(KernelProgramCache &Cache,
     return State == BS_Done || State == BS_Failed;
   });
 
+  if (WithBuildState->BuildResult.get()) {
```

Comment:
`WithBuildState` doesn't sound like a noun. If it is an actual build state, then you should probably drop `With`. 
https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
Done